github/packages.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+// Package represents a GitHub package.
+type Package struct {
+	ID             *int64           `json:"id,omitempty"`
+	Name           *string          `json:"name,omitempty"`
+	PackageType    *string          `json:"package_type,omitempty"`
+	HTMLURL        *string          `json:"html_url,omitempty"`
+	CreatedAt      *Timestamp       `json:"created_at,omitempty"`
+	UpdatedAt      *Timestamp       `json:"updated_at,omitempty"`
+	Owner          *User            `json:"owner,omitempty"`
+	PackageVersion *PackageVersion  `json:"package_version,omitempty"`
+	Registry       *PackageRegistry `json:"registry,omitempty"`
+}
+
+func (p Package) String() string {
+	return Stringify(p)
+}
+
+// PackageVersion represents a GitHub package version.
+type PackageVersion struct {
+	ID                  *int64          `json:"id,omitempty"`
+	Version             *string         `json:"version,omitempty"`
+	Summary             *string         `json:"summary,omitempty"`
+	Body                *string         `json:"body,omitempty"`
+	BodyHTML            *string         `json:"body_html,omitempty"`
+	Release             *PackageRelease `json:"release,omitempty"`
+	Manifest            *string         `json:"manifest,omitempty"`
+	HTMLURL             *string         `json:"html_url,omitempty"`
+	TagName             *string         `json:"tag_name,omitempty"`
+	TargetCommitish     *string         `json:"target_commitish,omitempty"`
+	TargetOID           *string         `json:"target_oid,omitempty"`
+	Draft               *bool           `json:"draft,omitempty"`
+	Prerelease          *bool           `json:"prerelease,omitempty"`
+	CreatedAt           *Timestamp      `json:"created_at,omitempty"`
+	UpdatedAt           *Timestamp      `json:"updated_at,omitempty"`
+	PackageFiles        *[]PackageFile  `json:"package_files,omitempty"`
```

Comment:
I think the type should be `[]*PackageFile`. Wait for somebody else to confirm it though.

Reply:
Yes, deifinitely. Since a slice is already a reference type, we don't typically use pointers to slices.
Please change this to `[]*PackageFile`.